### PR TITLE
Taran is very great

### PR DESCRIPTION
**Pull request by moogoesmeow0**
## Overview

This PR will do things

## Activity Finished

I finished activity 1

## Tests Ran

I ran no tests

## Additional Notes

yayayayayaya

### CODE REVIEW

File: src/main/java/frc/robot/constants/DriveConstants.java [hunk 1]
```diff
@@ -13,12 +13,12 @@
...
   // TODO 1.1.1: Set these
-  public static final int LEFT_MOTOR_1_ID = -1;
+  public static final int LEFT_MOTOR_1_ID = 11;
+  public static final int RIGHT_MOTOR_1_ID = 6;
   public static final int LEFT_MOTOR_2_ID = -1;
-  public static final int RIGHT_MOTOR_1_ID = -1;
   public static final int RIGHT_MOTOR_2_ID = -1;
 
-  // TODO 6.1.3: Add FF and PID constants
+  // TODO 6.1.3: Add FF and PID constants pooplear
```
Comment (Kyle-Eldridge) [inline src/main/java/frc/robot/constants/DriveConstants.java]: Why did you add this?

File: src/main/java/frc/robot/subsystems/Drivetrain.java [hunk 2]
```diff
@@ -1,8 +1,15 @@
 package frc.robot.subsystems;
 
+import java.net.CacheRequest;
+
```
Comment (Kyle-Eldridge) [inline src/main/java/frc/robot/subsystems/Drivetrain.java]: Remove unused imports.

File: src/main/java/frc/robot/subsystems/Drivetrain.java [hunk 3]
```diff
@@ -46,7 +55,7 @@ public Drivetrain() {
   public void periodic(){
     // TODO 2.2.5: Update odometry
 
-    // TODO 1.2.2: Call tankDrive()
+    tankDrive(Robot.driver.getRawLeftTranslation() * .25, Robot.driver.getRawRightTranslation() * .25);
```
Comment (Kyle-Eldridge) [inline src/main/java/frc/robot/subsystems/Drivetrain.java:54]: This works, but you might want to move the *.25 to the tankDrive method instead of here so it works in the next activity.

**Review by Kyle-Eldridge — CHANGES_REQUESTED** [review]
This is mostly good, but you should make a few small changes. You also need to update from main.

File: src/main/.wpilib/wpilib_preferences.json
Comment (johaj10) [inline src/main/.wpilib/wpilib_preferences.json:1]: Just FYI, it looks like this was added automatically by the WPILib extension. Should be fine.

File: src/main/java/frc/robot/subsystems/Drivetrain.java (hunk 3 above)
Comment (johaj10) [reply src/main/java/frc/robot/subsystems/Drivetrain.java:54]: Yeah it's probably better to multiple by .25 when you're setting the motor power in the `tankDrive` method.

File: src/main/java/frc/robot/constants/DriveConstants.java (hunk 1 above)
Comment (moogoesmeow0) [reply src/main/java/frc/robot/constants/DriveConstants.java]: typo
but kinda funny

File: src/main/java/frc/robot/subsystems/Drivetrain.java (hunk 2 above)
Comment (moogoesmeow0) [reply src/main/java/frc/robot/subsystems/Drivetrain.java]: fixed

File: src/main/java/frc/robot/subsystems/Drivetrain.java (hunk 3 above)
Comment (moogoesmeow0) [reply src/main/java/frc/robot/subsystems/Drivetrain.java:54]: fixed

**Review by Kyle-Eldridge — APPROVED** [review]
